common/app/model/content/Atoms.scala

```diff
@@ -515,7 +518,37 @@ object ReviewAtom {
 }
 
 object StoryQuestionsAtom {
-  def make(atom: AtomApiAtom): StoryQuestionsAtom = StoryQuestionsAtom(atom.id, atom, atom.data.asInstanceOf[AtomData.Storyquestions].storyquestions)
+  def make(atom: AtomApiAtom): StoryQuestionsAtom = StoryQuestionsAtom(
+    atom.id,
+    atom,
+    atom.data.asInstanceOf[AtomData.Storyquestions].storyquestions
+  )
+
+  def isClosed(question: StoryQuestionsAtom): Boolean = {
```

Comment:
This should be used to filter out closed storyquestions and not worry about them later.

Reply:
You can update [`extract`](https://github.com/guardian/frontend/blob/0ea1950923d9765618118d57084ab48ce2b6469f/common/app/model/content/Atoms.scala#L209) to do that for you:
```
def extract[T](
  atoms: Option[Seq[AtomApiAtom]], 
  extractFn: AtomApiAtom => T,
  filter: AtomApiAtom => Boolean = const(false)): Seq[T] = {
    try {
      atoms.getOrElse(Nil).filterNot(filter).map(extractFn)
    } catch {
      case e: Exception =>
        logException(e)
        Nil
    }
}
```
and then
```scala
val storyquestions = extract(atoms.storyquestions, atom => { StoryQuestionsAtom.make(atom) }, StoryQuestionsAtom.isClosed)
```